pkg/ui/key_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"src.elv.sh/pkg/eval/vals"
 )
 
-var kTests = []struct {
+var keyTests = []struct {
```

Comment:
The `golint` warning on this is hilarious, it thinks that the leading `k` in `kTests` is the private member marker used by some C++ and C# code...

But renaming this to `keyTests` is wrong because these are test cases of the `K` function. A lesser evil is to rename this to `KTests`.

Reply:
Done.